public/index.html

```diff
@@ -1,284 +1,455 @@
+          >
+            <div class="flex px-4">
+              <div
+                class="text-5xl font-bold text-[#01FF6B] sm:text-8xl"
+                x-text="counters.days"
+              ></div>
+              <div class="rotate-180 py-2 text-sm">
+                <div class="rotate-90 px-1">dni</div>
+              </div>
             </div>
-          </div>
-          <div class="flex flex-wrap justify-center gap-4">
-            <div class="flex flex-col items-center px-4">
-              <div class="text-4xl font-bold" x-text="counters.hours"></div>
-              <div class="flex text-sm py-2">
-                <div class="px-1">godzin</div>
+            <div class="flex px-4">
+              <div class="text-5xl font-bold sm:text-8xl" x-text="counters.hours"></div>
+              <div class="rotate-180 py-4 text-sm">
+                <div class="rotate-90 px-1">godzin</div>
               </div>
             </div>
-            <div class="flex flex-col px-4">
-              <div class="text-4xl font-bold" x-text="counters.minutes"></div>
-              <div class="flex text-sm py-2">
-                <div class="px-1">minut</div>
+            <div class="flex px-4">
+              <div class="text-8xl font-bold" x-text="counters.minutes"></div>
+              <div class="rotate-180 py-4 text-sm">
+                <div class="rotate-90 px-1">minut</div>
               </div>
             </div>
-            <div class="flex flex-col items-center px-4">
-              <div class="text-4xl font-bold" x-text="counters.seconds"></div>
-              <div class="flex text-sm py-2">
-                <div class="px-1">sekund</div>
+            <div class="flex px-4">
+              <div class="text-8xl font-bold" x-text="counters.seconds"></div>
+              <div class="rotate-180 py-4 text-sm">
+                <div class="rotate-90 px-1">sekund</div>
               </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
-
-    <template x-if="meetup">
-      <section class="mt-12 p-4">
-        <div class="mx-auto max-w-7xl text-white flex-col">
-          <div class="hidden lg:block absolute right-0 -mt-[12em] md:-mr-[40em] xl:-mr-[24em] border border-[84px] border-[#01FF6B] size-[56em] rounded-full z-0"></div>
-          <div class="flex flex-col gap-6 ml-0 sm:ml-36 font-bold text-center lg:text-left">
-            <div class="relative text-5xl sm:text-6xl xl:text-9xl lg:mt-6">
-              <span x-text="meetup.date"></span>
-              <div class="lg:hidden" x-text="meetup.year"></div>
-              <div class="absolute">
-                <div class="-mt-12 bg-[#01FF6B] h-3 w-64 -ml-72"></div>
-              </div>
+      <div
+        class="mx-9 mb-32 mt-[90px] grid max-w-screen-2xl grid-cols-1 gap-7 text-white md:mt-44 lg:mx-3 lg:grid-cols-3 xl:px-14 2xl:mx-auto 2xl:px-0"
+      >
+        <div class="rounded-3xl border border-[#ad92e0] p-8">
```

Comment:
This border color is used multiple times in the file, I think it would be better to create custom color for this in `tailwind.config.js`

Reply:
Replaced hash color to custom from tailwind config.